packages/dds/tree/src/test/simple-tree/tree.spec.ts

```diff
@@ -58,6 +59,45 @@ describe("class-tree tree", () => {
 		view.initialize({ stuff: ["a", "b"] });
 	});
 
+	it("root access leak", () => {
+		const config = new TreeViewConfiguration({ schema: Canvas });
+		const tree = factory.create(
+			new MockFluidDataStoreRuntime({ idCompressor: createIdCompressor() }),
+			"tree",
+		);
+		const view = tree.viewWith(config);
+		view.initialize({ stuff: [] });
+		const _unused = view.root;
+		const context = view.getView().context;
+		const countBefore = context.withAnchors.size + context.withCursors.size;
```

Comment:
I don't have much context on the implementation, but I'm assuming you're validating the sum here rather than that both match for some reason. That might be worth a small comment somewhere.